### ISSUE TITLE
Improve the "getting started" flow

### ISSUE THREAD

**Issue by failuer** [alloverse/open-planning#94]
## Introduction/Info

- [x] Make sure to clarify that references to a “visor app” is the same thing as the Alloverse desktop app. Or just make sure to use the same nomenclature.
- [x] Clarify that the Alloverse app is "a window to the Alloverse". "Start the app, connect to the sandbox place and leave it running in the background. This is your "window to the Alloverse, and a way to see and interact with your app as you build, run and iterate on it.
- [x] Fix broken formatting of ´components´ TOC
- [x] Be super clear that you don't need a headset - most people are probably gonna do the tutorial using the desktop app.
- [x] Make sure the guide is always presuming the user is using the sandbox. 
- [x] Clarify that the Alloverse app and the terminal running the todo app are separate. "Open the Alloverse app, click "connect", type "sandbox" into the input field.
- [x] ~Be clear that you don't _need_ to understand Penlight.~
- [x] Make sure all (external) links open in new tabs.
- [x] Remove `./allo/assist help` as a _requirement_ for running the app. We can just say it's a nice-to-have.
- [x] Write the tutorial from the perspective of using the desktop app - not a headset. Mention early that a headset can absolutely be used, but for the rest of the guide, focus on the desktop scenario (for example; "when pressing the input field, this shows a virtual keyboard (unless you have a hardware keyboard connected)")
- [x] "If your app ever disappears - take a look around in the Alloverse to make sure you're not just looking in the wrong direction. If you still can't see it, it has probably crashed. Check out the terminal from which you run it - there should be error messages to help you figure out what went wrong."
- [x] "Officially" rename the "Alloverse (Desktop) App" to "Visor", or "AlloVisor" (ergo, change all references in the getting started guide).

## Coding
- [x] Clarify where the layout function should be pasted (below the `_init` function)
- [x] Add clarifying output when `run` is successful, i.e. "App started successfully! Open the Alloverse Visor and connect to [URL] to see it in action!".
- [x] Remove debug output “incompatible track” & “destroyed” messages. 
- [x] As for what to do with the rest of the [DEBUG] messages, create a ticket for Nevyn: https://github.com/alloverse/allonet/issues/33
- [x] Clarify that after having added the first code block, there is an "add todo" button - _but you can't press it yet_ (because its handler hasn't been added). It doesn't matter that the next sentence is “now lets implement the button behaviour”, - the user hasn't gotten there yet.
- [x] Clarify "Fire off a quick `./allo/assist run`" to also include the URL to the sandbox after so the command is complete.
- [x] Fix the input field looking janky - the border is ~10cm above (y-axis) the field where you can type.
- [ ] Where applicable, document the code snippets we're asking the user to copy/paste. That way there isn't as much need for tabbing back and forth. Also, it would match nicely with the currently very well documented dummy project stub.
- [ ] The "Pen" pattern is a bit verbose and has been obsoleted by a vertical stack - so use that instead. 
- [x] Clarify the section where "add the checkmark to the assets by referring to it under the quit button texture you added previously. 

## Other
- [x] Test & fix broken links in AlloUI documentation
- [x] The sandbox should have a "testing ground" skybox/environment. Partly as feedback that yes, indeed, the user have successfully connected to a place and yes, it is indeed the sandbox they wanted to end up in. Think "Street Fighter 4 practice arena".
- [x] ~~It's not clear that "Components" is not the same as "AlloUI lua/c#". Clarify by adding "API": "AlloUI API (lua/c#)" and move it to the top. "Components" and "Architecture" are more complicated and not required to get started building an app, so they can afford to be further down in the left hand menu hierarchy.~~

## When everything else is done...
- [ ] Update all screenshots to exactly match every step of the guide.

**Comment by failuer** [comment]
![Image](https://user-images.githubusercontent.com/3473998/175001425-5ed3fc25-ec34-4cb9-8b55-3ae331af72e1.png)

Pretty scary output for just having successfully launched your first TODO app 😅 

@nevyn, what of this output can we remove? What is still useful feedback?

What can we add:
**App started successfully! Open the Alloverse Visor and connect to [URL] to see it in action!**

**Comment by nevyn** [comment]
@failuer either we disable output of debug messages, or we remove message 3+4. 

Love the addition of the friendly output!

**Comment by failuer** [comment]
Sandbox environment inspiration:

![Image](https://user-images.githubusercontent.com/3473998/176914129-ecd23023-2229-428d-b95e-5e91754cf7ed.png)

**Comment by failuer** [comment]
Blocked by not understanding race conditions & markAsDirty updates. Don't know if my layout code is incorrect or it just /displays/ incorrectly.

**Comment by failuer** [comment]
So much has been done here, i'd like to push the improvements that's been finished.

Thus, I created a new issue with the single blocking issue that's left: https://github.com/alloverse/alloverse.github.io/issues/3
